README.md

```diff
@@ -1 +1,155 @@
-# App-Blog
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230139676-ae31a16f-65ee-4f6a-bc17-ebcb856336ff.png)

**[OPTIONAL]**

It would be very great if you make all your commit messages more meaningful and capitalize them because I saw your commit is not capitalized 😨 Kindly consider them making capitalized and meaningful to make your pull request professional 👍